### PR TITLE
Bepro 1625 big token names breakes frontend layout

### PR DESCRIPTION
**Pull request by MarcusviniciusLsantos**

### CODE REVIEW

**Review by moshmage — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Pull Request Document
```
Comment (moshmage) [inline .github/PULL_REQUEST_TEMPLATE.md]: rebase your branch as this change is not from this PR

File: components/custom-network/network-contract-settings.tsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ export default function NetworkContractSettings() {
       disabled: totalNetworkToken.gt(0),
       helperText: totalNetworkToken.gt(0) ? t("custom-network:oracle-exchange-rate.unable-to-change", {
         amount: formatNumberToNScale(totalNetworkToken.toFixed()),
-        symbol: networkTokenSymbol
+        symbol: networkTokenSymbol?.length > 6 ? `${networkTokenSymbol.slice(0, 6)}...` : networkTokenSymbol
```
Comment (moshmage) [inline components/custom-network/network-contract-settings.tsx:90]: set this lenght (6) as a constant

File: components/profile/my-network-settings/governance-settings.tsx [hunk 3]
```diff
@@ -54,7 +54,8 @@ export default function GovernanceSettings({
     forcedNetwork,
   } = useNetworkSettings();
 
-  const networkTokenSymbol = forcedNetwork?.networkToken?.symbol;
+  const symbol = forcedNetwork?.networkToken?.symbol
+  const networkTokenSymbol = symbol?.length > 12 ? `${symbol.slice(0, 12)}...` : symbol
```
Comment (moshmage) [inline components/profile/my-network-settings/governance-settings.tsx:58]: set this value (12) as a constant (...SMALL ....LARGE)

File: components/token-symbol.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { CSSProperties } from "react";
```
Comment (moshmage) [inline components/token-symbol.tsx:1]: this component can be moved to "common input components" like folder

**Review by moshmage — CHANGES_REQUESTED** [review]
add the min with values to constants so we can easily change them

**Review by moshmage — APPROVED** [review]

**Review by vhcsilva — APPROVED** [review]

**Review by vhcsilva — APPROVED** [review]

**Pull request merged**